src/api_client/constants.js

```diff
@@ -15,3 +15,4 @@
 export const GET = 'GET';
 export const POST = 'POST';
 export const PUT = 'PUT';
+export const MAX_RETRY_COUNT = 3;
```

Comment:
I think it makes sense to indicate what the retry count is for - how about `API_RECONNECT_MAX_RETRY_COUNT` ?